src/redux/selectors/index.js

```diff
@@ -81,3 +84,24 @@ export const timelinePropsSelector = createSelector(
     maximumDate: timelineItems[timelineItems.length - 1]?.timelineDate,
   }),
 );
+
+export const patientAgeAtResourcesSelector = createSelector(
+  [patientSelector, timelineItemSelector],
+  (patient, timelineItems) => {
+    if (!patient) {
+      return {};
+    }
+    return timelineItems.reduce((acc, { id, timelineDate }) => {
+      const birthDate = patient?.birthDate;
+      const resourceDate = format(new Date(timelineDate), 'yyyy-MM-dd');
+      const ageAtResourceDate = intervalToDuration({
+        start: parse(birthDate, 'yyyy-MM-dd', new Date()),
```

Comment:
`const start = parse(birthDate, 'yyyy-MM-dd', new Date()),`

this could be computed once, outside of the loop?